.github/workflows/build_dockers.yml

```diff
@@ -0,0 +1,161 @@
+      run: |
+        pip install -r core_team_ci_tools/github_artifacts/requirements.txt --user
+
+    - name: Download mimirsbrunn package
+      run: |
+        mimirsbrunn_package="debian-package-release.zip"
+        python core_team_ci_tools/github_artifacts/github_artifacts.py \
+            -o hove-io \
+            -r mimirsbrunn \
+            -t ${{secrets.access_token_github}} \
+            -w release.yml \
+            -a $mimirsbrunn_package \
+            --output-dir .
+        unzip -qo $mimirsbrunn_package
+        rm -f $mimirsbrunn_package
+
+    - name: Download cosmogony2cities package
+      run: |
+        cosmogony2cities_package="package-${{env.debian_version}}.zip"
+        python core_team_ci_tools/github_artifacts/github_artifacts.py \
+            -o hove-io \
+            -r cosmogony2cities \
+            -t  ${{secrets.access_token_github}} \
+            -w build_package.yml \
+            -a $cosmogony2cities_package \
+            --output-dir .
+        unzip -qo $cosmogony2cities_package
+        rm -f $cosmogony2cities_package
+
+    - name: Restore ccache
+      uses: hendrikmuhs/ccache-action@v1.2
+      with:
+        key: ${{env.debian_version}-package
+        max-size: 2000M
+        save: ${{ github.event_name == 'push' }}
+
+    - name: Checkout navitia
+      uses: actions/checkout@v3
+      with:
+        submodules : recursive
+        path: navitia
+        fetch-depth: 0
+
+    - name: Build navitia packages
+      working-directory: navitia
+      # Will build navitia-*.deb packages in folder ../
+      run: |
+        DEB_BUILD_OPTIONS=nocheck dpkg-buildpackage -b
+
+
+    # see https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable
+    - name: Choose dev navitia tag
+      if: github.event_name == 'push' && github.ref == 'refs/heads/dev'
+      run: |
+        echo "navitia_tag=dev" >> $GITHUB_ENV
+
+    - name: Choose release navitia tag
+      if: startsWith(github.ref, 'refs/tags/')
+      run: |
+        cd navitia
+        version=$(git describe)
+        echo "building version $version"
+        echo "navitia_tag=$version" >> $GITHUB_ENV
+
+
+    - name: Create dockers images and push them
+      run: |
+        docker build -f navitia/docker/${{env.debian_version}}/Dockerfile-master -t navitia/master:${{env.navitia_tag}} .
+
+        components='jormungandr kraken tyr-beat tyr-worker tyr-web instances-configurator mock-kraken eitri'
+        for component in $components; do
+            echo "*********  Building $component ***************"
+            docker build -t navitia/$component:${{env.navitia_tag}} -f  navitia/docker/${{env.debian_version}}/Dockerfile-${component} .
+
+            # add latest tag if navitia_tag != dev
+            if [[ "${{env.navitia_tag}}" != "dev" ]]; then
+                docker tag navitia/$component:${{env.navitia_tag}}  navitia/$component:latest
+            fi
+        done
```

Comment:
I'm wondering, would it be feasible to create a Github Actions `matrix` to parallelize the build of these components? Might be too much for this PR at the moment though :shrug: